src/components/ChatLog.jsx

```diff
@@ -0,0 +1,39 @@
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry.jsx';
+
+const ChatLog = (props) => {
+  const messageComponents = props.entries.map(message => {
```

Comment:
I also prefer to use a variable to hold my mapped components! Just a heads, you might see the pattern where the mapping is done in the JSX object. Some developers prefer that since they feel like it helps them understand the overall structure of the component.